src/RatkoR/Crate/Connectors/Connector.php

```diff
@@ -29,7 +29,10 @@ class Connector extends BaseConnector implements ConnectorInterface {
      */
     public function createConnection($dsn, array $config, array $options)
     {
-        return new PDO($dsn, null, null, $options);
+        $username = $config['username'] ?: null;
```

Comment:
If user doesn't have `username` or  `password` field set this line fails with:

```
ErrorException: Undefined index: username
```

If you use `??` instead of `?:` things work as expected. Can you change it please.

Reply:
Yes, for sure. I forgot that short ternary syntax doesn't like when the index doesn't exist. I've pushed the update.